math.asm

```diff
@@ -163,3 +164,59 @@ end:    INX
     sta MSB+1,x
     inx
     rts
+
+    +BACKLINK "invert", 6
+INVERT
+    lda MSB, x
+    eor #$ff
+    sta MSB, x
+    lda LSB, x
+    eor #$ff
+    sta LSB,x
+    rts
+
+    +BACKLINK "negate", 6
+NEGATE
+    jsr INVERT
+    jmp ONEPLUS
+
+    +BACKLINK "abs", 3
+ABS
+    lda MSB,x
+    bmi NEGATE
+    rts
+
+DABS_STAR
+    lda MSB,x
+    eor MSB+1,x
+    pha
+    jsr ABS
+    inx
+    jsr ABS
+    dex
+    jsr U_M_STAR
+    pla
+    rts
+
+    +BACKLINK "*", 1
+    jsr DABS_STAR
+    inx
+    and #$ff
+    bmi NEGATE
+    rts
+
+    +BACKLINK "dnegate", 7
+DNEGATE
+    jsr INVERT
+    inx
+    jsr INVERT
+    dex
+    lda #1
+    ldy #0
+    jsr pushya
+    jmp M_PLUS
```

Comment:
I don't like this approach to `D1+`.  I'm working on the division right now and it's preventing some refactoring.

If you're okay with it, I'll continue division in this pull request?

Reply:
Changed in this pull request anyway.